common/tools/eslint-plugin-azure-sdk/package.json

```diff
@@ -50,7 +50,7 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "unit-test:node": "mocha --require source-map-support/register --require ts-node/register --timeout 10000 --full-trace tests/**/*.ts",
+    "unit-test:node": "cross-env VITE_CJS_IGNORE_WARNING=true vitest",
```

Comment:
@maorleger we are seeing these because the CJS support for `vite` as noted [here](https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated)